app/controllers/course_user_data_controller.rb

```diff
@@ -174,7 +176,7 @@ def update
 
   action_auth_level :destroy, :instructor
   def destroy
```

Comment:
destroy / destroyConfirm don't actually seem to be routes that are currently used, might be worth removing.